cub/cub/agent/agent_adjacent_difference.cuh

```diff
@@ -107,7 +107,7 @@ struct AgentDifference
   DifferenceOpT difference_op;
   OffsetT num_items;
 
-  __device__ __forceinline__ AgentDifference(TempStorage &temp_storage,
+  _CCCL_DEVICE _CCCL_FORCEINLINE AgentDifference(TempStorage &temp_storage,
```

Comment:
There are some formatting issues here, that said I tried running clang-format and the changes are extensive. Thoughts?

I am not sure how much time we want to spend with formatting here. Thoughts?

Reply:
I'd be fine leaving this change unformatted, since we still have lots of other unformatted parts. I think, ultimately, we want to do a big bang PR that will format the rest and pick up these lines as well.